### DISCUSSION TITLE
Compiler inference for constants

### DISCUSSION THREAD

**Discussion by andyrub18** [dotnet/csharplang#8860] [Language Ideas] [answered]
I'd like to propose compiler inference for constants in C#. The fact is constants need to be initialized so I don't see why it is necessary to precise the type when declaring a constant.
The same way it is normal to declare a variable with `var myVariable = "A variable"` it should also be possible to do `const myConstant = "A constant"`

**Comment by HaloFour** [comment, marked as answer]
See: #8663
